__tests__/index.test.js

```diff
@@ -81,20 +81,31 @@ describe('/api/articles', () => {
 })
 
 describe('/api/articles/:id', () => {
-  it('GET:200 issues a response where res.body.article returns the matching article', async () => {
-    const res = await request(app).get('/api/articles/1').expect(200)
-    const { article } = res.body
-
-    expect(article).toEqual({
-      id: 1,
-      author: 'butter_bridge',
-      title: 'Living in the shadow of a great man',
-      topic: 'mitch',
-      body: 'I find this existence challenging',
-      created_at: '2020-07-09T20:11:00.000Z', // hard to replicate programatically
-      vote_count: 100,
-      img_url:
-        'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+  describe('GET:200 issues a response where res.body.article returns', () => {
+    it('the matching article', async () => {
+      const res = await request(app).get('/api/articles/1').expect(200)
+      const { article } = res.body
+
+      expect(article).toMatchObject({
+        author: 'butter_bridge',
+        title: 'Living in the shadow of a great man',
+        topic: 'mitch',
+        body: 'I find this existence challenging',
+        vote_count: 100,
+        img_url:
```

Comment:
There's a property missing on this, `created_at` should also be in the response object. 
I find that, while it is possible to match the date format from the test data, using `expect.any(String)` will suffice for the value of this property